### PR TITLE
Update legend table

### PR DESCRIPTION
**Pull request by aloptrbl**
at info

### CODE REVIEW

File: src/components/Menu.js [hunk 1]
```diff
@@ -2,9 +2,86 @@ import { html } from 'htm/preact';
...
+tr:nth-child(n+2) > td:nth-child(2)
+{
+
+}
+td{
+  text-align: center;
+  padding: 15px;
+}
+.circle{
+width: 30px;
```
Comment (thepassle) [inline src/components/Menu.js:58]: Can we make these 20px to be consistent with the circle we use elsewhere?

File: src/components/Menu.js [hunk 2]
```diff
@@ -34,6 +111,49 @@ const renderMenu = (menuItem) => {
...
+          <${Expandable}
+            toggle=${'Legend'}
+            detail=${html`
+              <p>The map shows two parameters for territories:</p>
+              <ol>
+              <li>If the country is under any type of lockdown</li>
+              <li>If there are covid cases in the country</li>
+              </ol>
+            <table class=${styles}>
+<tr>
```
Comment (thepassle) [inline src/components/Menu.js:123]: Can you run `npm run format` to format this code?

**Comment by thepassle** [comment]
Can we add a little more margin between the table and the list?

image

**Comment by thepassle** [comment]
The colors look off in darkmode, can you fix this?

image

File: src/components/Menu.js [hunk 3]
```diff
@@ -34,6 +111,49 @@ const renderMenu = (menuItem) => {
...
+<th></th>
+<th>Lockdown</th>
+<th>COVID-19</th>
+</tr>
+<tr>
+<td><div class="blue circle"/></td>
```
Comment (thepassle) [inline src/components/Menu.js]: This has no semantic value for users with a screen reader, can you look into this? Let me know if you need help :)

File: src/components/Menu.js [hunk 4]
```diff
@@ -2,9 +2,86 @@ import { html } from 'htm/preact';
...
+.green {
+  background: #92c47c;
+}
+
+width: 100%;
+border-collapse: collapse; 
+
+`;
+
+const circle = css`
```
Comment (thepassle) [inline src/components/Menu.js]: if this is empty we can remove it

File: src/components/Menu.js [hunk 5]
```diff
@@ -34,6 +111,49 @@ const renderMenu = (menuItem) => {
...
+<td>${close}</td>
+</tr>
+<tr>
+<td><div class="red circle"/></td>
+<td>${trues}</td>
+<td>${trues}</td>
+</tr>
+<tr>
+<td><div class="default circle"/></td>
+<td>No Data <p class="note">(please contribute)</p></td>
```
Comment (thepassle) [inline src/components/Menu.js]: Maybe we can add a link to https://docs.google.com/forms/d/e/1FAIpQLSfDWe2qlzUnd3e-YYspMzT9adUswDEYIdJMb7jz7ule34-yiA/viewform here to make it easy for people to contribute

File: src/components/Menu.js (hunk 3 above)
Comment (aloptrbl) [reply src/components/Menu.js]: I dont understand for this part. What do you mean by no semantic value for users with a screen reader? What should I do here?

Comment (thepassle) [reply src/components/Menu.js]: Some users with a vision disablity may use a screen reader to use the application. Currently, this element has no value to relay any information to the screen reader (meaning that the screen reader will just ignore this element, and people with a vision disability will not have access to this information). You can solve this by adding a `aria-label="blue"` to these divs

**Comment by thepassle** [comment]
Hi @aloptrbl , just checking in to see if there has been any updates on this. Do you need any more help or assistance? 🙂 Please feel free to reach out

**Comment by aloptrbl** [comment]
@thepassle working on it right now. btw do i need create new branch instead?

**Comment by thepassle** [comment]
No need, they turned out to be a few minor additions to your code, so I merged it already. Thanks for your contribution!

**Comment by aloptrbl** [comment]
thanks @thepassle , hope can learn more from you. 😬

**Pull request merged**